src/views/month/single-day.php

```diff
@@ -40,7 +40,7 @@
 	<div class="tribe-events-viewmore">
 		<?php
 
-			$view_all_label = sprintf( _n( 'View 1 %1$s', 'View All %2$s %3$s', $day['total_events'], 'tribe-events-calendar' ), $events_label_singular, $day['total_events'], $events_label_plural );
+			$view_all_label = sprintf( _n( 'View 1 %s', 'View All %s %s', $day['total_events'], 'tribe-events-calendar' ), $events_label_singular, $day['total_events'], $events_label_plural );
```

Comment:
The `%s` just grabs the params in order starting with the first one (`$events_label_singular`) regardless of which string `_n()` chooses.